library/core/src/iter/traits/peekable.rs

```diff
@@ -0,0 +1,25 @@
+#[unstable(feature = "peekable_iterator", issue = "132973")]
+/// Iterators which inherently support `peek()` without needing to be wrapped by a `Peekable`.
```

Comment:
Since the signature of the required method differs from `Peekable::peek`, I’d change the description to avoid mentioning that method specifically:

```suggestion
/// Iterators which inherently support peeking without needing to be wrapped by a `Peekable`.
```